src/main/java/org/jabref/gui/collab/DatabaseChangeMonitor.java

```diff
@@ -62,10 +63,22 @@ public DatabaseChangeMonitor(BibDatabaseContext database,
             }
         });
 
+        LibraryTab.DatabaseNotification notificationPane = libraryTab.getNotificationPane();
         addListener(changes -> notificationPane.notify(
                 IconTheme.JabRefIcons.SAVE.getGraphicNode(),
                 Localization.lang("The library has been modified by another program."),
-                List.of(new Action(Localization.lang("Dismiss changes"), event -> notificationPane.hide()),
+                List.of(new Action(Localization.lang("Dismiss changes"), event -> libraryTab.getNotificationPane().hide()),
+                        new Action(Localization.lang("Accept changes"), event -> {
+                            // Perform all accepted changes
+                            NamedCompound ce = new NamedCompound(Localization.lang("Merged external changes"));
+                            for (DatabaseChangeViewModel change : changes) {
+                                if (change instanceof EntryChangeViewModel || change.isAccepted()) {
```

Comment:
I think, this line comes from org.jabref.gui.collab.ChangeDisplayDialog#ChangeDisplayDialog - line 85 following. I think, this `if` can be removed, since these changes are all external changes, which are all accepted.